src/frontend/src/Components/shared/Filters.js

```diff
@@ -150,7 +156,20 @@ export default function Filters(props) {
 
       {open &&
         <div className="filters">
+          <div className="filters-title__container">
           <h4 className="filters-title">{textOverride ? textOverride : 'Layer filters'}</h4>
+          {
+            (mapLayers && mapLayers.current.routeLayer && mapLayers.current.routeLayer.rendered) 
+            &&
+            <div>
+              <label className="filters-title__notification">
+                Current routes
+                <OverlayTrigger placement="top" overlay={tooltipCurrentRoutes}>
+                  <button className="filters-title__tooltip-info" aria-label="current routes tooltip" aria-describedby="tooltipCurrentRoutes">?</button>
```

Comment:
This should be using the circle-question icon from fontawesome instead of just a "?"